### ISSUE TITLE
Propose alternative Created Timestamp handling for OpenMetrics Text Format

### ISSUE THREAD

**Issue by bwplotka** [prometheus/prometheus#13537]
### Proposal

Following our [protobuf change](https://github.com/prometheus/client_model/pull/66) and proposal to update [the text format with native histograms](https://github.com/prometheus/proposals/pulls), it would be epic to consider designing and implementing created timestamp feature to Prometheus text format (or better to OpenMetrics Text in place of current _created series).

We consider this to be a work for a GSoC mentorship project.

**Comment by NeerajGartia21** [comment]
Hi @bwplotka, I'm interested in participating in GSoC'24 and contributing to this project. Could you please provide guidance on any prerequisites I should fulfil beforehand? 
Recently I opened a PR for `UTF-8 parser for UI for the new syntax`.

**Comment by kartikaysaxena** [comment]
Hey @NeerajGartia21 are prev mentees allowed to participate in GSoC?

**Comment by NeerajGartia21** [comment]
Yes @kartikaysaxena , afaik one can participate for the second time in GSoC but not in LFX Mentorship Program.

**Comment by kartikaysaxena** [comment]
Hey @NeerajGartia21 but the rules state otherwise https://developers.google.com/open-source/gsoc/faq#what_are_the_eligibility_requirements_for_participation

**Comment by Maniktherana** [comment]
hey @bwplotka I'm also looking to contribute to Prometheus and participate in GSoC. Looking forward to helping out!

**Comment by Thisausername99** [comment]
Hi @bwplotka, I am interested in contributing for this project for GSoC 2024 program. Beside reviewing the historical thread of the underlying project, do you have any recommend resources that I could refer to beforehand? Furthermore, do we have a dedicated communication channel for this project?

**Comment by kevinwielander** [comment]
Hi @bwplotka, I am also intrested in controbuting for this project for GSoC 2024. Looking forward to a positive reply.

**Comment by zhifanl** [comment]
Hi @bwplotka, I am interested in contributing to this project for GSoC 2024. 
I am a software engineering student of the University of Calgary with a background in Cloud Computing, DevOps, Software Development, Open Source contribution. I also have three internship experiences in North America. 

I just read the design doc and I understand what you are trying to achieve. I am wondering in order to contribute to improving and optimizing the proposed method of adding a zero-value sample with the created timestamp (synthetic zero) to the Prometheus Time Series Database? are there any other related documents I can read?

**Comment by bwplotka** [comment]
There's a progress for this thanks to @Maniktherana 

* https://github.com/prometheus/proposals/pull/43
* https://github.com/prometheus/prometheus/pull/16033

**Comment by Maniktherana** [comment]
> There's a progress for this thanks to @Maniktherana 
> 
> * https://github.com/prometheus/proposals/pull/43
> * https://github.com/prometheus/prometheus/pull/16033

Also https://github.com/prometheus/common/pull/760

**Comment by bboreham** [comment]
Hello from the bug-scrub!
Upstream issue is https://github.com/prometheus/OpenMetrics/issues/285; we should close this and await that outcome.
